dashboard/test/controllers/concerns/pd/workshop_filters_test.rb

```diff
@@ -71,6 +71,94 @@ class FakeController < ::ApplicationController
     @controller.load_filtered_ended_workshops
   end
 
+  test 'filter_workshops default' do
+    @controller.filter_workshops @workshop_query
+  end
+
+  test 'filter_workshops with state' do
+    expects(:in_state).with('Not Started', error_on_bad_state: false)
+    params state: 'Not Started'
+    @controller.filter_workshops @workshop_query
```

Comment:
Everything is stubbed here so nothing useful is returned. These test cases are verifying that the expected queries are made via. the `expects`

Or do you mean for the default test on line 74? In that case, with no expectation, any filter will fail the test. I can add a comment explaining this, and/or do some more explicit validation.